cmd/minishift/cmd/completion.go

```diff
@@ -31,15 +31,21 @@ const longDescription = `
 	Outputs minishift shell completion for the given shell (bash or zsh)
```

Comment:
@tylerauerbeck I think you messed up with git squash :smile:  

Technically this PR should have only one file change `.circleci/config.yml`.